tests/validation.js

```diff
@@ -1,37 +1,31 @@
 import test from 'ava';
 import validation from '../lib/validation';
 
-const input = {
+const empty = {
   target: {
     name: 'checkbox',
-    value: 'foo bar baz',
-  },
-  all: {
-    checkbox: 'foo bar baz',
+    value: '',
   },
 };
 
-const settings = {
+const input = {
   target: {
-    empty: false,
+    name: 'checkbox',
+    value: 'foo bar baz',
   },
   all: {
-    checkbox: {
-      empty: false,
-    },
+    checkbox: 'foo bar baz',
   },
 };
 
+const settings = {};
+
 
 // Valid input
-test('valid input', t => {
-  t.true(validation(input, settings), 'Valid input returns true');
+test('empty input', t => {
+  t.true(validation(empty, settings), 'empty input returns true');
 });
 
-// Invalid input
-test('validate correct input', t => {
-  const ip = input;
-  ip.target.value = '';
-
-  t.is(validation(ip, settings), 'checkbox cannot be left blank!', 'Return string if not valid');
+test('valid input', t => {
+  t.true(validation(input, settings), 'Any input returns true');
```

Comment:
Ish. I am perhaps guilty of overtesting, but I don't mind. The bug to be fixed was about empty inputs failing, so we should make sure to test for that as well as a basic input.